### ISSUE TITLE
Wrong encoding of content in partial widget pages #2

### ISSUE THREAD

**Issue by rkeitel** [KenticoDevTrev/PartialWidgetPage#12]
Characters - like ä, ö, ü, ß in German - are displayed as "ÃƒÂ¶" or something else. If I show the same content in a normal widget, which is not added to a partial widget page, the characters are correctly encoded.

I have a container in the root with employees. On the home page I have a hierarchy of partial widget pages:
![image](https://user-images.githubusercontent.com/52745832/84758748-278a9380-afc6-11ea-9960-6cb62dbefef9.png)

The deepest partial widget page "Employee Highlight (Anja)" has a link (NodeGUID) to the selected employee to show the content. The name is displayed correctly:
![image](https://user-images.githubusercontent.com/52745832/84758608-f611c800-afc5-11ea-926b-bc95c9dd9590.png)

On "Top One-Column B" there are a partial page widget (top) and a default widget (bottom). On the default widget, the name is displayed correctly, but not on the partial page widget:
![image](https://user-images.githubusercontent.com/52745832/84759062-894afd80-afc6-11ea-80ee-231e4de7b0fc.png)
![image](https://user-images.githubusercontent.com/52745832/84758608-f611c800-afc5-11ea-926b-bc95c9dd9590.png)

On the home page, where "Top One-Column B" is added as partial page widget, the name is displayed incorrectly, for both!
Top one (partial page widget) from the partial widget page:
![image](https://user-images.githubusercontent.com/52745832/84759978-ba77fd80-afc7-11ea-9b5a-dd12f0bdd1ce.png)
Bottom one (default widget) from the partial widget page:
![image](https://user-images.githubusercontent.com/52745832/84759062-894afd80-afc6-11ea-80ee-231e4de7b0fc.png)

The deeper the hierarchy of partial widget pages is the often the encoding is wrong.

**Comment by KenticoDevTrev** [comment]
I was unable to replicate, when i added your special characters it properly displayed the right characters on the partial widget page.  There may be more going on with this though.  It's not hard to tweak the encoding settings, but it's hard for me to know what to tweak it to.  Perhaps you can fork your branch and see what needs to be done that fixes it in your case?

**Comment by rkeitel** [comment]
Did you try it with my approach (container with content only pages) or with a text editor?

**Comment by KenticoDevTrev** [comment]
The text was in a rich text widget and pulled in via the html helper in
another page.

**Comment by rkeitel** [comment]
What happens if you add a static text with umlauts to the page whereon you add the widget?

**Comment by KenticoDevTrev** [comment]
Can you paste me the html output of the page that gets pulled in with the
partial widget so I can try to replicate?

**Comment by rkeitel** [comment]
```
<section class="employee-content">
  <div class="jumbotron">
    @if (Model.Employee == null)
    {
      <p>@Html.LocalizeExt("Staff.Employee.LinkInfo")</p>
    }
    else if (!Model.Employee.HasValues)
    {
      <p>@Html.LocalizeExt("Staff.Employee.DataInfo")</p>
    }
    else
    {
      <div class="row">
        @if (!string.IsNullOrEmpty(Model.Employee.ImagePath))
        {
          <div class="col-auto">
            <img alt="@Model.Employee.FirstName @Model.Employee.LastName" src="@Url.ImageUrlExt(Model.Employee.ImagePath, new MaxWidthOrHeightSizeConstraint(122))">
          </div>
        }
        <div class="col">
          <h1>@Model.Employee.FirstName @Model.Employee.LastName äöüß</h1>
        </div>
      </div>
    }
  </div>
</section>
```

**Comment by KenticoDevTrev** [comment]
Thanks, having it hard coded in there allowed me to replicate.  I adjusted the Encoding to UTF8, that fixed the issue, the AJAX loading one did not show this issue, only the server side.  I updated both my nuget package and the custom one i made for you, however going forward i probably won't be updating your package so you'll need to generate your own packages going forward, but hopefully no more issues.
